sphinx/ext/autodoc/importer.py

```diff
@@ -127,11 +129,13 @@ def load_module(self, name):
 @contextlib.contextmanager
 def mock(names):
     # type: (List[str]) -> Generator
+    importer = None
     try:
         importer = _MockImporter(names)
         yield
     finally:
-        importer.disable()
+        if importer is not None:
```

Comment:
Why did you check `importer here`? Did you mean instantiation of `_MockImporter` might be failed?

Reply:
Yes, I put the explanation in the git commit message:

https://github.com/sphinx-doc/sphinx/commit/fe24d5ca608589fcda3d6602ec3f742214c99064

Reply:
Oh, sorry. I'd overlooked it. I understand.